app/utils/well-known-ids.js

```diff
@@ -1,2 +1,4 @@
 export const MALE_ID = '5ab0e9b8a3b2ca7c5e000028';
 export const FEMALE_ID = '5ab0e9b8a3b2ca7c5e000029';
+export const FRACTIE_FORM_ID = 'fractie';
+export const FORM_DEFINITION_MODEL = 'form-definition';
```

Comment:
this may be driving it a little far... model names are normally just used as a string in ember